ClientApp/src/app/Flight.ts

```diff
@@ -0,0 +1,14 @@
+import {Airport} from './Airport';
+import {Time} from '@angular/common';
```

Comment:
What do you need these imports for?

Reply:
I've deleted Airport import as it's not needed anymore, but Time is required to set datatype Time to property flight_length